pkg/opentelemetry/config.go

```diff
@@ -39,6 +39,18 @@ type Config struct {
 	// ExportInterval configures the intervening time between metrics exports
 	ExportInterval types.NullDuration `json:"exportInterval" envconfig:"K6_OTEL_EXPORT_INTERVAL"`
 
+	// Headers in W3C Correlation-Context format without additional semi-colon delimited metadata (i.e. "k1=v1,k2=v2")
+	Headers null.String `json:"headers" envconfig:"K6_OTEL_HEADERS"`
+
+	// TLSInsecureSkipVerify disables verification of the server's certificate chain
+	TLSInsecureSkipVerify null.Bool `json:"tlsInsecureSkipVerify" envconfig:"K6_OTEL_TLS_INSECURE_SKIP_VERIFY"`
+	// TLSCertificate is the path to the certificate file (rootCAs) to use for the
```

Comment:
```suggestion
	// TLSCertificate is the path to the certificate file (rootCAs) to use for the
```
to use for the ... ? it sounds like we are missing a part of it

I would also add the concept that the certs from the system are used if insecure is set to false and the cert is not passed.

Reply:
Yeah, it's the same for all the certificate/key configuration variables, good catch!

Reply:
Good catch :+1:  Fixed with the latest force push